app/views/content_items/_case_study_a.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= content_for :title, "#{@content_item.page_title} - #{t("content_item.format.#{@content_item.document_type}", count: 1)}" %>
```

Comment:
Like the comment above, I think we could try to rename these partials to something other than a/b, so it's clearer what they render. What do you think?